doc/connectivity/bluetooth/api/mesh/proxy.rst

```diff
@@ -9,6 +9,72 @@ mesh network through GATT. The Proxy feature is only compiled in if the
 controlled by the :ref:`bluetooth_mesh_models_cfg_srv`, and the initial value
 can be set with :c:member:`bt_mesh_cfg_srv.gatt_proxy`.
 
+Nodes with the Proxy feature enabled can advertise with Network Identity and Node Identity,
+which is controlled by the :ref:`bluetooth_mesh_models_cfg_cli`.
+
+The GATT Proxy state indicates if the Proxy feature is supported.
+
+Private Proxy
+*************
+
+A node supporting the :ref:`bluetooth_mesh_models_priv_beacon_srv` can advertise with
+Private Network Identity and Private Node Identity types, which is controlled by the
+:ref:`bluetooth_mesh_models_priv_beacon_cli`. Private Beacons provide privacy by periodically
+randomizing the beacon input data. By advertising with this set of identification types,
+the node is allowing its Proxy Client to connect to the network over GATT while maintaining the
+privacy of the network. This functionality is called Private Proxy. For the Private Proxy
+functionality to be supported, the Proxy feature, Private Network Identity advertising, and
+Mesh Private beacons must be supported by the node.
```

Comment:
I think the last 2 sentences aren't needed. What needs to be supported by the node and what Private Proxy gives is explained above. This is more like a spec statement, but for a user to enable the support of this feature it is enough to instantiate Private Beacon Server model and enable GATT Proxy support (which is explained in the section above).
```
This functionality is called Private Proxy. For the Private Proxy functionality to be supported, the Proxy feature, Private Network Identity advertising, and Mesh Private beacons must be supported by the node.
```